### PR TITLE
feat: add basic axios interceptor

### PR DESCRIPTION
**Pull request by viniciuslacerda0**
## Description

Add Axios Interceptor 
Add Axios config

## Related Issues

[#24](https://trello.com/c/QXiCqAuN/24-adicionar-estrutura-para-o-axios)

## Progress

- [x] Axios Interceptor
- [x] Axios config

## How to test it

Go to Create Account and try to add an invalid or valid user, look at console if request is showing

## Visual reference

image

## Change logs

Axios Interceptor
Axios config

### CODE REVIEW

**Review by gustavolbs — CHANGES_REQUESTED** [review]
Please, add QA Steps and what is expect from those tests (since it's not a UI test).
Please, create a ticket at Trello and add it to your PR.

File: src/index.tsx [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 
 import './styles.css';
 
+setupInterceptorsTo(axios);
+
+axios.defaults.baseURL = '';
+axios.defaults.headers.common.Authorization = '';
+axios.defaults.headers.post['Content-Type'] =
+  'application/x-www-form-urlencoded';
+
```
[the remark this answers is not in the document]
Comment (viniciuslacerda0) [reply src/index.tsx]: I need to create a global instance that changes the axios config. I created a function in the `services/index` and call it on `src/index`

**Review by gustavolbs — APPROVED** [review]

**Review by gustavolbs — APPROVED** [review]
I do not agree in sharing request and response at console, since we do not want to expose sensitive data. From the perspective from the code quality, I think it's okay, but you should discuss with our PO @marianasn before merging this.

**Comment by viniciuslacerda0** [comment]
> I do not agree in sharing request and response at console, since we do not want to expose sensitive data. From the perspective from the code quality, I think it's okay, but you should discuss with our PO @marianasn before merging this.

I agree with you, I`ll be adding the errors in a logger system (like Sentry), so it is just a structure in the moment, I can remove the console loggers since it was done only to test things up

**Pull request merged**